consensus/istanbul/core/types.go

```diff
@@ -69,94 +66,6 @@ func (s State) Cmp(y State) int {
 	return 0
 }
 
-const (
```

Comment:
Moved to `consensus/istanbul/types.go`. TBH not sure why.

Reply:
I think it needed to be used in `backend` and `core` and there was some issue with circular dependencies